edge/evtstreams/cpu2evtstreams/service.sh

```diff
@@ -112,17 +112,10 @@ while true; do
       #echo "avg: $json"
 
       if [[ "$PUBLISH" == "true" ]]; then
-        if [[ -n "$EVTSTREAMS_CERT_FILE" ]]; then
-          # Send data to ICP Event Streams
-          echo "echo $json | kafkacat -P -b $EVTSTREAMS_BROKER_URL -X api.version.request=true -X security.protocol=sasl_ssl -X sasl.mechanisms=PLAIN -X sasl.username=token -X sasl.password=$EVTSTREAMS_API_KEY -X ssl.ca.location=$EVTSTREAMS_CERT_FILE -t $EVTSTREAMS_TOPIC"
-          echo "$json" | kafkacat -P -b $EVTSTREAMS_BROKER_URL -X api.version.request=true -X security.protocol=sasl_ssl -X sasl.mechanisms=PLAIN -X sasl.username=token -X sasl.password=$EVTSTREAMS_API_KEY -X ssl.ca.location=$EVTSTREAMS_CERT_FILE -t $EVTSTREAMS_TOPIC
-          checkrc $? "kafkacat" "continue"
-        else
-          # Send data to IBM Cloud Event Streams
-          echo "echo $json | kafkacat -P -b $EVTSTREAMS_BROKER_URL -X api.version.request=true -X security.protocol=sasl_ssl -X sasl.mechanisms=PLAIN -X sasl.username=$EVTSTREAMS_USERNAME -X sasl.password=$EVTSTREAMS_PASSWORD -t $EVTSTREAMS_TOPIC"
-          echo "$json" | kafkacat -P -b $EVTSTREAMS_BROKER_URL -X api.version.request=true -X security.protocol=sasl_ssl -X sasl.mechanisms=PLAIN -X sasl.username=$EVTSTREAMS_USERNAME -X sasl.password=$EVTSTREAMS_PASSWORD -t $EVTSTREAMS_TOPIC
-          checkrc $? "kafkacat" "continue"
-        fi
+        # Send data to Event Streams
+        echo "echo $json | kafkacat -P -b $EVTSTREAMS_BROKER_URL -X api.version.request=true -X security.protocol=sasl_ssl -X sasl.mechanisms=PLAIN -X sasl.username=token -X sasl.password=$EVTSTREAMS_API_KEY -X ssl.ca.location=$EVTSTREAMS_CERT_FILE -t $EVTSTREAMS_TOPIC"
+        echo "$json" | kafkacat -P -b $EVTSTREAMS_BROKER_URL -X api.version.request=true -X security.protocol=sasl_ssl -X sasl.mechanisms=PLAIN -X sasl.username=token -X sasl.password=$EVTSTREAMS_API_KEY -X ssl.ca.location=$EVTSTREAMS_CERT_FILE -t $EVTSTREAMS_TOPIC
```

Comment:
When this is sending to the public cloud, is it really ok with the argument `-X ssl.ca.location=` ? (because `$EVTSTREAMS_CERT_FILE` is empty)

Reply:
Yeah I made sure to have both `EVTSTREAMS_CERT_FILE` and `EVTSTREAMS_CERT_ENCODED` completely unset when I tested staging to public cloud, icp to public cloud, and finally icp to icp event streams. And there was no issue. The last bit of the kafkacat command just left it completely blank 

`... sasl.password=nVYHj_zIZP36imSfizpu3EwIKaqMvQfPR-dCKdZg7jR1 -X ssl.ca.location= -t cpu2evtstreams`

Reply:
ok, cool, just checking